weblate/templates/translate.html

```diff
@@ -469,7 +469,7 @@ <h5>
             {% if user_can_ignore_check %}
               <a href="{% url 'js-ignore-check' check_id=check.id %}" data-check="{{ check.id }}" data-dismiss-all="{% url 'js-ignore-check-source' check_id=check.id %}" class="btn btn-warning check-dismiss check-dismiss-single">{% trans "Dismiss" %}</a>
               {% if user_can_edit_source %}
-                <label class="dismiss-all"><input type="checkbox" name="dismiss-all-{{ check.id }}" /> {% trans "For all languages" %}</label><span style="display:inline-block;height:24px;vertical-align:super;margin-left:6px;" title="{% trans "Selecting this option will set the corresponding \"ignore check\" flag" %}">{% icon "info.svg" %}</span>
+                <label class="dismiss-all"><input type="checkbox" name="dismiss-all-{{ check.id }}" /> {% trans "For all languages" %}</label><span class="info-tooltip" title="{% trans "Selecting this option will set the corresponding &#34;ignore check&#34; flag" %}">{% icon "info.svg" %}</span>
```

Comment:
You can use single quotes to avoid escaping here:

```suggestion
                <label class="dismiss-all"><input type="checkbox" name="dismiss-all-{{ check.id }}" /> {% trans "For all languages" %}</label><span class="info-tooltip" title="{% trans 'Selecting this option will set the corresponding "ignore check" flag.' %}">{% icon "info.svg" %}</span>
```

Reply:
ok I was trying to figure out how to correctly escape, I wasn't having much luck with backslashes. Thanks for the input, implementation complete.